src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Button from '../../../../common/Button'
+import { StyledCourseCard } from '../../../../common/CommonHTML'
+import { mockedAuthorsList } from '../../../../constants'
+import getDuration from '../../../../helpers/getCourseDuration'
+
+export default function CourseCard(props) {
+  const {
+    title, description, creationDate, duration, authors
+  } = props.item
+  const formatCreation = creationDate.split('/').join('.')
```

Comment:
Following from the above comment, `createDate` is passed as one of the props, but we use it and modify the format.
We should consider the rule, do only one thing in one time.
In the code, the `CourseCard` does at least two things:
1. Build the component
2. Transform data format(`formatCreation`)

We should de-couple them.

we could pass the `formatCreation` as creationDate

Reply:
The `authors` should be changed to `authorIDs` from the usage.